language/stdlib/transaction_scripts/mint_lbr.move

```diff
@@ -1,11 +1,73 @@
 script {
 use 0x1::LibraAccount;
+// prover deps:
+use 0x1::Signer;
+use 0x1::Option;
 
 /// Mint `amount_lbr` LBR from the sending account's constituent coins and deposits the
 /// resulting LBR into the sending account.
+///
+/// # Events
```

Comment:
This documentation seems the same as that in stdlib.rs.  Is one automatically generated from the other?

Reply:
The comments in stdlib.rs are auto generated from here(mint_lbr.move).